src/Interfaces/VTK.jl

```diff
@@ -7,9 +7,31 @@ using Reexport
 using WriteVTK
 
 using ..Meshes
+using ..Nodes
+using ..StructuralAnalyses
 
 @reexport import WriteVTK: vtk_grid
+export write_vtks
 
-# Add code here.
+function write_vtks(states_sol, filename)
+    nodes_mat = node_matrix(states_sol.analysis.s.mesh)
+    connec_mat = connectivity(states_sol.analysis.s.mesh)
+    num_elem = length(states_sol.analysis.s.mesh.elements)
+    cells = [MeshCell(VTKCellTypes.VTK_TETRA, connec_mat[e]) for e in 1:num_elem]
+
+    pdata = displacements(states_sol)[end]
+    vtk_grid(filename, nodes_mat, cells) do vtk
+        vtk["Displacements", VTKPointData()] = pdata
+    end
+
+    #=
+    s = states_sol.analysis.s
+    for n in nodes(s)
+        # Get the displacements for all coordinates at the last time point.
+        data = [last(c) for c in displacements(states_sol, n)]
+        vtk_point_data(filename, data, "Displacements")
+    end
+    =#
```

Comment:
remove?

Reply:
done in [cfcd24c](https://github.com/ONSAS/ONSAS.jl/pull/414/commits/cfcd24ced1944a2f76fecff1ad4b4dcd46901a7a)